tests/kafkatest/services/security/minikdc.py

```diff
@@ -107,6 +107,12 @@ def start_node(self, node):
         cmd = "for file in %s; do CLASSPATH=$CLASSPATH:$file; done;" % core_libs_jar
         cmd += " for file in %s; do CLASSPATH=$CLASSPATH:$file; done;" % core_dependant_test_libs_jar
         cmd += " export CLASSPATH;"
+        # exports sun.security.krb5 avoiding Java 17 error:
+        # Exception in thread "main" java.lang.IllegalAccessException:
+        # class kafka.security.minikdc.MiniKdc cannot access class sun.security.krb5.Config
+        # (in module java.security.jgss) because module java.security.jgss does not export
+        # sun.security.krb5 to unnamed module @24959ca4
```

Comment:
I don't think we have to include the full error message, it's fine to say it fixes an `IllegalAccessException` due to MiniKdc's usage of sun.security.krb5.Config.

Reply:
Okay, I will change it :)